src/index.js

```diff
@@ -11,6 +11,8 @@ var tap = function(func){
 }
 
 var cronofy = function(config){
+  this.config = config;
+
   this.accountInformation = function(){
```

Comment:
this sets it on the prototype due to using new? TIL :D
You might want to assert here that the function is being called using new